### PR TITLE
HOMS-238 Use env file in homs's apps

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -63,22 +63,22 @@ services:
...
+      - activiti:$BPM_HOST
     volumes:
-      - /etc/hydra/homs/homs_configuration.yml:/opt/homs/config/homs_configuration.yml
-      - /etc/hydra/homs/imprint.yml:/opt/homs/config/imprint.yml
-      - /etc/hydra/homs/hbw.yml:/opt/homs/config/hbw.yml
-      - /etc/hydra/homs/database.yml:/opt/homs/config/database.yml
-      - /etc/hydra/homs/bpm.yml:/opt/homs/config/bpm.yml
-      - /etc/hydra/homs/sources.yml:/opt/homs/config/sources.yml
-      - /etc/hydra/homs/secrets.yml:/opt/homs/config/secrets.yml
+      - /Users/mxsmirnoff/dev/homs/config/homs_configuration.yml:/opt/homs/config/homs_configuration.yml
```
Comment (dolfinus) [inline docker-compose.yml]: Should be `/etc/hydra`

File: config/homs_configuration.yml.sample [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    token: null
+    sign_in_as: user@example.com
+
+link_to_manual: "http://hydra-oms.com/docs"
+
+use_imprint: true
+
+logs:
+  color: false
+  development:
```
Comment (dolfinus) [inline config/homs_configuration.yml.sample]: Log configuration is not used now, everything writes to stdout

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ v1.5.0 [unreleased]
 -   [#179](https://github.com/latera/homs/pull/179) Replace `jquery-ui` tooltip with `popper.js`.
 -   [#184](https://github.com/latera/homs/pull/184) Add `base_url` option to main config provides custom prefix for all paths in app. Fixes [#148](https://github.com/latera/homs/issues/148).    
 -   [#185](https://github.com/latera/homs/pull/185) Speed up docker build.
-
+-   [#188](https://github.com/latera/homs/pull/188) Use env file in homs's apps
```
[the remark this answers is not in the document]
Comment (sugarfree1) [reply CHANGELOG.md]: Change text to: `Use .env file in docker-compose`

File: docker-compose.dev.oracle.yml [hunk 4]
```diff
@@ -21,33 +21,33 @@ services:
...
     restart: always
     environment:
-      PGDATA: /var/lib/postgresql/data/homs
+      PGDATA: $HOMS_DB_PATH
       POSTGRES_DB: $HOMS_DB_NAME
       POSTGRES_USER: $HOMS_DB_USER
       POSTGRES_PASSWORD: $HOMS_DB_PASSWORD
       TZ: Europe/Moscow
     volumes:
       - $HOMS_PATH/data/homs/postgresql:/var/lib/postgresql/data
```
Comment (TimAle) [inline docker-compose.dev.oracle.yml]: `:$HOMS_DB_PATH`.
Same with other db data volumes.

Comment (mxsmirnoff) [reply docker-compose.dev.oracle.yml]: Don't understand what is wrong.

File: README.md [hunk 5]
```diff
@@ -48,53 +48,29 @@ The prefered way to install HOMS is to use docker
   cp config/secrets.yml.sample /etc/hydra/homs/secrets.yml
   ```
 
-5. Create `.env` near `docker-compose.yml` file and fill it up with applications credentials. Default values:
+5. Copy your (or default) env file to your project directory:
```
Comment (sugarfree1) [inline README.md]: Let it be `.env` file.

Comment (sugarfree1) [reply README.md]: ...to your project's directory.

File: config/hbw.yml.sample [hunk 6]
```diff
@@ -45,5 +45,5 @@ hbw:
     endpoint: 'http://127.0.0.1:9000'
     access_key_id: <%= ENV['MINIO_ACCESS_KEY'] %>
     secret_access_key: <%= ENV['MINIO_SECRET_KEY'] %>
-    bucket: 'bucket_name'
+    bucket: <%= ENV['MINIO_BUCKET_NAME'] %>
     region: 'us-east-1'
```
Comment (sugarfree1) [inline config/hbw.yml.sample]: I think this should be in `.env` file to.

File: docker-compose.test.yml [hunk 7]
```diff
@@ -5,34 +5,32 @@ services:
     hostname: postgres-homs
     container_name: postgres-homs
     restart: always
+    env_file:
+    - "homs.env.test"
```
Comment (sugarfree1) [inline docker-compose.test.yml]: `.env.test`

File: homs.env.sample [hunk 8]
```diff
@@ -0,0 +1,28 @@
+ HOMS_PATH=/path/to/homs
```
Comment (sugarfree1) [inline homs.env.sample]: `/dir/with`

**Comment by TimAle** [comment]
Also add `.env` to `.gitignore` please.

**Comment by mxsmirnoff** [comment]
Updated

**Comment by TimAle** [comment]
Please
image

**Pull request merged**